dist/backdraft.js

```diff
@@ -845,11 +845,19 @@ _.extend(Plugin.factory, {
     ',
     sFilterListTemplate : '\
       <li><label><input class="list" id="value" type="checkbox" name="{{= attr }}" \
-         value="{{= options }}" /> <%= options %></label></li> \
+         value="{{= options }}" /> {{= options }}</label></li> \
+    ',
+    sActionButtonTemplate : '\
+      <button class="btn btn-primary btn-sm {{= klass}}" name="button" type="submit" title="">{{= caption}}</button>\
+    ',
+    sFilterButtonTemplate : '\
```

Comment:
could be renamed?  I had trouble thinking of a better name, let me know what you think.